lib/src/widgets/adw/new/scaffold.dart

```diff
@@ -10,17 +10,13 @@ class AdwScaffold extends StatefulWidget {
   /// Remove this when ViewSwitcher becomes adaptive
   final Widget? bottomNavigationBar;
 
-  AdwScaffold({
+  const AdwScaffold({
     Key? key,
     required this.body,
     this.flapController,
-    Widget? drawer,
+    this.drawer,
```

Comment:
What sense makes this parameter here? We already have the `AdwFlap` widget to control this use case. I think this is a bit confusing and could be deleted no problem.

In fact it was confusing enough that the `AdwSidebar` was added twice because of that.

Reply:
We need it as a drawer for mobile screen as using it as a flap will squeeze the content.

Reply:
@jesusrp98 you're pointed at (IMO) the most awkward part of the current implementation of libadwaita for now.
If you look at the example app, there is a drawer for when the sidebar is folded, but also a sidebar for the rest of the time. They are both a copy-paste of each other's where the only thing that changes if the callback for the `onSelected` method where the drawer needs to be popped...

I can't think of a good way to solve that situation yet :(

Reply:
Ok, I'll restore those changes. Thanks for the explanation. I'll also create an issue to disccuss how we could improve that, because you're, that is a bit weird.